api_backend_server/tests/integration_tests/test_activities_apis.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import requests
+from collections import namedtuple
+
+@pytest.fixture(scope="module")
+def credentials():
```

Comment:
Function name should be changed - it creates dummy activity now instead of credentials